### ISSUE TITLE
save model and test model

### ISSUE THREAD

**Issue by Joll123** [liorsidi/ViolenceDetection_CNNLSTM#9]
Thanks for your work.
I save model by model.save(model_path+"_model.h5"), and then restore model by  keras.models.load_model('./results/_cnn_train-retrain_optimizer-RMSprop_learning_rate-0d0001_batch_size-2_dataset-violentflow_epochs-50_cnn-ResNet50_initial_weights-glorot_uniform_lstm-ConvLSTM2D_dropout-0d0_seq_len-20_model.h5'). 
Choice 21 consecutive pictures in the video stream, normalize each picture, and then (Image_t - Image_t+1) to get 20 pictures as the model input.
Finally, use  model.predict() predict result, but  I find that the performances are not good.

**Comment by liorsidi** [comment]
@Joll123 did you used your own dataset for the test?
